framework/decode/vulkan_replay_options.h

```diff
@@ -47,6 +47,7 @@ struct VulkanReplayOptions : public ReplayOptions
     bool                         skip_failed_allocations{ false };
     bool                         omit_pipeline_cache_data{ false };
     bool                         remove_unsupported_features{ false };
+    bool                         colorspace_fallback{ false };
```

Comment:
Our options are frequently "verb noun", so I suggest "use_colorspace_fallback" or "use_fallback_colorspace" for this variable and the CL option, etc

Reply:
I renamed the CL option and the variable as suggested.